test/testcafe/spec/BYOL_spec.js

```diff
@@ -1,22 +1,33 @@
 import BYOLPageObject from '../framework/page-objects/BYOLPageObject';
-import countryFooLanguageBundle from '../../../playground/mocks/labels/json/country_foo';
-import loginFooLanguageBundle from '../../../playground/mocks/labels/json/login_foo';
-import xhrAuthenticatorEnrollDataPhone from '../../../playground/mocks/data/idp/idx/authenticator-enroll-data-phone';
+import xhrAuthenticatorEnrollDataPhone from '../../../playground/mocks/data/idp/idx/authenticator-enroll-data-phone.json';
```

Comment:
Did we intentionally drop automatic resolution for `*.json`?

Reply:
I don't think we dropped the automatic resolution for `*.json` because I see other spec tests that do not have the extension and still works. I am checking with Lester to see if this was done for a particular reason.

Reply:
Response from Lester on this: 

IMHO we should change it to add the .json ext everywhere. Module resolution gets unnecessarily messy/ambiguous. I see the .json ext preserved in most default configs from frameworks, but Gen2 has it set up in webpack to resolve as module. It has the additional benefit of being able to navigate to the file in VS Code (and other IDE's) if it has the extension

Reply:
Okay great, that makes sense to me